source/exe/main.cc

```diff
@@ -32,12 +32,17 @@ int main(int argc, char** argv) {
 
 #ifdef ENVOY_HOT_RESTART
   // Enabled by default, except on OS X. Control with "bazel --define=hot_restart=disabled"
-  const std::string shared_mem_version = Envoy::Server::SharedMemory::version();
+  const Envoy::OptionsImpl::HotRestartVersionCb hot_restart_version_cb =
+      [](uint64_t max_num_stats, uint64_t max_stat_name_len) {
+        return Envoy::Server::SharedMemory::version(max_num_stats, max_stat_name_len);
+      };
 #else
-  const std::string shared_mem_version = "disabled";
+  const Envoy::OptionsImpl::HotRestartVersionCB hot_restart_version_cb = [](uint64_t, uint64_t) {
```

Comment:
This broke the MacOS build. `HotRestartVersionCB` -> `HotRestartVersionCb`

Reply:
Sorry.  I'll post a fix.

Reply:
https://github.com/envoyproxy/envoy/pull/1825